### PR TITLE
GUI: Add new panel for Training Schedule (Reattempt)

### LINKED ISSUE #64: Modify GUI to support training sessions
Additional window to display training sessions - medium
Minor improvements (display of student information) - high

### PR DESCRIPTION
**Pull request by kerkpy**
Fixes #64

### CODE REVIEW

File: src/main/java/seedu/address/ui/TrainingCard.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.student.Student;
+
+/**
+ * An UI component that displays information of a {@code Student}.
```
Comment (sudogene) [inline src/main/java/seedu/address/ui/TrainingCard.java:13]: An UI component that displays information of a ... 

{@code Training} ?
{@code TrainingSession} ?
{@code TrainingSchedule} ?

File: src/main/java/seedu/address/ui/TrainingListPanel.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.student.Student;
+
+/**
+ * Panel containing the list of students.
```
Comment (sudogene) [inline src/main/java/seedu/address/ui/TrainingListPanel.java:14]: Panel containing the list of training/training sessions/training schedules.

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -33,25 +33,35 @@
...
-          </padding>
-          <StackPane fx:id="studentListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
-        </VBox>
+            <SplitPane dividerPositions="0.5" prefHeight="160.0" prefWidth="200.0" VBox.vgrow="ALWAYS">
+              <items>
+                                            <VBox fx:id="studentList" prefWidth="340.0" styleClass="pane-with-border">
+                <padding>
+                  <Insets bottom="10" left="10" right="10" top="10" />
+                </padding>
+                <StackPane fx:id="studentListPanelPlaceholder" VBox.vgrow="ALWAYS" />
```
Comment (sudogene) [inline src/main/resources/view/MainWindow.fxml:53]: Just putting a note here for myself

**Review by sudogene — COMMENTED** [review]
Sweet, love it. Just need minor change to the header comments

Do we call it a Training, TrainingSession, or TrainingSchedule?

File: src/main/java/seedu/address/ui/TrainingCard.java (hunk 1 above)
Comment (kerkpy) [reply src/main/java/seedu/address/ui/TrainingCard.java:13]: We should call it Training, I think there would be no ambiguity what we would be referring to. I left the header comments there as a placeholder, would change it after the creation of a Training class. 

Thanks!

**Pull request merged**